### PR TITLE
radosgw-admin: Create --secret-key alias for --secret

### PR DESCRIPTION
**Pull request by zhouyuan**
Fixes: #5821

Signed-off-by: Yuan Zhou <yuan.zhou@intel.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
nit pick: the typo in the title. =)

**Comment by zhouyuan** [comment]
@tchaikov Thanks! It's fixed now.

File: src/rgw/rgw_admin.cc [hunk 1]
```diff
@@ -1172,7 +1173,7 @@ int main(int argc, char **argv)
       access_key = val;
     } else if (ceph_argparse_witharg(args, i, &val, "--subuser", (char*)NULL)) {
       subuser = val;
-    } else if (ceph_argparse_witharg(args, i, &val, "--secret", (char*)NULL)) {
+    } else if (ceph_argparse_witharg(args, i, &val, "--secret", "--secrte-key", (char*)NULL)) {
       secret_key = val;
```
Comment (theanalyst) [inline src/rgw/rgw_admin.cc]: typo "--secret-key"

Comment (zhouyuan) [inline src/rgw/rgw_admin.cc]: @theanalyst thank you! fixed.

**Comment by theanalyst** [comment]
LGTM

**Comment by yehudasa** [comment]
@zhouyuan there's a merge conflict, I can't merge it, can you rebase it? Thanks

**Comment by zhouyuan** [comment]
@yehudasa rebased on the master. Thanks for the review!

**Comment by dachary** [comment]
@zhouyuan please ignore, this is a false negative ( http://tracker.ceph.com/issues/12340 ).

**Pull request merged**